src/components/Dashboard/TodoList.vue

```diff
@@ -366,13 +366,18 @@
                     </template>
                     <v-list class="actions__more-actions">
                       <v-list-item
+                        class="dissolution-drop-down"
+                        @mouseenter="hover = true"
```

Comment:
this one can be controlled by css only
like 
dissolution-drop-down:hover{
css
}